ui/page/privacy/manual_mixer_setup_page..go

```diff
@@ -0,0 +1,226 @@
+		Title("Unmixed account").
+		AccountSelected(func(selectedAccount *dcrlibwallet.Account) {}).
+		AccountValidator(func(account *dcrlibwallet.Account) bool {
+			wal := pg.Load.WL.MultiWallet.WalletWithID(account.WalletID)
+
+			// Imported and watch only wallet accounts are invalid to use as an unmixed account
+			accountIsValid := account.Number != load.MaxInt32 && !wal.IsWatchingOnlyWallet()
+
+			return accountIsValid
+		})
+
+	pg.backButton, pg.infoButton = components.SubpageHeaderButtons(l)
+
+	return pg
+}
+
+// ID is a unique string that identifies the page and may be used
+// to differentiate this page from other pages.
+// Part of the load.Page interface.
+func (pg *ManualMixerSetupPage) ID() string {
+	return ManualMixerSetupPageID
+}
+
+// OnNavigatedTo is called when the page is about to be displayed and
+// may be used to initialize page features that are only relevant when
+// the page is displayed.
+// Part of the load.Page interface.
+func (pg *ManualMixerSetupPage) OnNavigatedTo() {
+	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
+
+	pg.mixedAccountSelector.SelectFirstWalletValidAccount(pg.wallet)
+	pg.unmixedAccountSelector.SelectFirstWalletValidAccount(pg.wallet)
+}
+
+// Layout draws the page UI components into the provided layout context
+// to be eventually drawn on screen.
+// Part of the load.Page interface.
+func (pg *ManualMixerSetupPage) Layout(gtx layout.Context) layout.Dimensions {
+	body := func(gtx C) D {
+		page := components.SubPage{
+			Load:       pg.Load,
+			Title:      "Manual setup",
+			WalletName: pg.wallet.Name,
+			BackButton: pg.backButton,
+			Back: func() {
+				pg.PopFragment()
+			},
+			Body: func(gtx C) D {
+				return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+					gtx.Constraints.Min.X = gtx.Constraints.Max.X
+					return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+						layout.Flexed(1, func(gtx C) D {
+							return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									return pg.mixerAccountSections(gtx, "Mixed account", pg.mixedAccountSelector.Layout)
+								}),
+								layout.Rigid(func(gtx C) D {
+									return layout.Inset{Top: values.MarginPaddingMinus15}.Layout(gtx, func(gtx C) D {
+										return pg.mixerAccountSections(gtx, "Unmixed account", pg.unmixedAccountSelector.Layout)
+									})
+								}),
+								layout.Rigid(func(gtx C) D {
+									return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16, Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+										return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+											layout.Rigid(func(gtx C) D {
+												return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+													layout.Rigid(func(gtx C) D {
+														return pg.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
+													}),
+												)
+											}),
```

Comment:
The two layouts can merge. from what i see after building, this `Alignment: layout.Middle` is not needed.
```suggestion
										return layout.Flex{
										   Axis: layout.Horizontal,
										   Alignment: layout.Middle,
										 }.Layout(gtx,
											layout.Rigid(func(gtx C) D {
												return pg.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
											}),
```